index.html

```diff
@@ -80,16 +85,16 @@ <h4 class="desktop_show">Tonic</h4>
           
           <div class="language1 inline">
             <ul>
-              <li class="langu">html</li>
-              <li class="langu">css</li>
-              <li class="langu">javascript</li>
+              <li class="langu langu-a">html</li>
+              <li class="langu langu-b">css</li>
+              <li class="langu langu-c">javascript</li>
               <li class="langu2">html</li>
               <li class="langu2">css</li>
               <li class="langu2">javascript</li>
             </ul>
           </div>
         </div>
-        <button type="button" class="btn-one">See project</button>
+        <button type="button" class="btn-one btns">See project</button>
       </div></div>
       <div class="project-card card-flex">
```

Comment:
- [ ] Just as the previous Code Reviewer mentioned, all the project-card sections are supposed to be generated using Javascript. You can remove all the HTML code inside the section with class **work-section** and generate it using a for loop for example in javascript.  
The project requirements request that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads).

This way, adding projects in the future will be easy. You will only add it in one place - as another object in the project's array.

Reply:
Can I call you so you help me because i dont understand what i need to do

Reply:
yes sure

Reply:
ok ill call you in 10mins

Reply:
Im trying to call you on  zoom

Reply:
Done